gene/mutate.go

```diff
@@ -167,59 +174,87 @@ func Insertion(g *Gene, genomePos int, alt []dna.Base) (EffectPrediction, error)
 			}
 		}
 
-		//TODO: Effect Prediction WIP
-		//answer.CdnaPos, answer.CdnaDist, err = GenomicPosToCdna(g, genomePos + 1)
-		//frame := (cdnaPos + 1) % 3
-		//var currCodon dna.Codon
-		//
-		//if frame != 0 { // if insertion disrupts a codon, note the codon being changed
-		//	currCodon, err = CdnaPosToCodon(g, cdnaPos)
-		//	answer.AaRef = []dna.AminoAcid{dna.TranslateCodon(&currCodon)}
-		//}
+		answer.CdnaPos, answer.CdnaDist, err = GenomicPosToCdna(g, genomePos+1)
+		answer.CdnaPos-- // Subtract 1 to get base zero
+		frame := (cdnaPos + 1) % 3
```

Comment:
They are actually the same value. I could have done the following 

`_, answer.CdnaDist, err = GenomicPosToCdna(g, genomePos)`
`answer.CdnaPos = cdnaPos`

But since `GenomicPosToCdna` calculates the cDNA position anyways, I just used that output since it was along for the ride anyways.